src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```
@@ -103,11 +107,15 @@
 			Condition="exists('$(MSBuildThisFileDirectory)..\Uno.UI.SourceGenerators.csproj')">
 
 		<ItemGroup >
-			<_UnoAnalyzerExclusion Include="$(MSBuildThisFileDirectory)..\bin\$(Configuration)\netstandard2.0\Microsoft.CodeAnalysis.*.dll" />
-			<_UnoAnalyzerExclusion Include="$(MSBuildThisFileDirectory)..\bin\$(Configuration)\netstandard2.0\System.*.dll" />
+			<_UnoAnalyzerExclusion1 Include="$(MSBuildThisFileDirectory)..\bin\$(Configuration)\netstandard2.0\Microsoft.CodeAnalysis.*.dll" />
+			<_UnoAnalyzerExclusion1 Include="$(MSBuildThisFileDirectory)..\bin\$(Configuration)\netstandard2.0\System.*.dll" />
+			<_UnoAnalyzerExclusion2 Include="$(MSBuildThisFileDirectory)..\..\Uno.UI.InternalSourceGenerators\bin\$(Configuration)\netstandard2.0\Microsoft.CodeAnalysis.*.dll" />
```

```suggestion
			<_UnoAnalyzerExclusionInternal Include="$(MSBuildThisFileDirectory)..\..\Uno.UI.InternalSourceGenerators\bin\$(Configuration)\netstandard2.0\Microsoft.CodeAnalysis.*.dll" />
```

same for the one below?